build.sbt

```diff
@@ -4,15 +4,15 @@ import uk.gov.hmrc.DefaultBuildSettings
 
 lazy val appName: String = "pact-broker-frontend"
 ThisBuild / majorVersion := 1
-ThisBuild / scalaVersion := "2.13.12"
+ThisBuild / scalaVersion := "3.3.3"
 
 lazy val root = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtDistributablesPlugin)
   .settings(ScoverageSettings())
   .settings(
     scalacOptions ++= Seq(
-      "-Werror",
-      "-Wconf:src=routes/.*:s"
+//      "-Xfatal-warnings"
```

Comment:
I would add a comment here saying why we are commenting and when we can uncomment